modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CustomViewsControls.js

```diff
@@ -15,6 +16,30 @@ import FrontendDataSetContext from '../../FrontendDataSetContext';
 import ViewsContext from '../../views/ViewsContext';
 import {VIEWS_ACTION_TYPES} from '../../views/viewsReducer';
 
+const DEFAULT_VIEW_ID = 'DEFAULT_VIEW';
+
+const CustomViewsControlsTrigger = React.forwardRef(
+	({triggerLabel, viewUpdated, ...otherProps}, ref) => (
```

Comment:
[Interesting API](https://github.com/liferay/clay/blob/master/packages/clay-core/src/picker/Picker.tsx#L57-L63) . @matuzalemsteles is there a reason to require a `ref`? For seemingly the same purpose, a component prop, we used [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSetContext.tsx#L71) `React.ComponentType<any>`.

Reply:
Hi @markocikos, In fact, this type is wrong, we just have to `React.ForwardRefExoticComponent`, we need the `ref` to calculate the menu positioning.